.github/workflows/dependabot-automerge.yaml

```diff
@@ -0,0 +1,21 @@
+# Automatically merge Dependabot PRs upon approval by leaving
+# a comment on Dependabot's pull-request.
+
+name: Automerge Dependabot PR
+on:
+  pull_request_review:
+    types: [submitted]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  comment:
+    if: ${{ github.event.review.state == 'approved' && github.event.pull_request.user.login == 'dependabot[bot]' }}
+    runs-on: ubuntu-latest
+    steps:
+    - name: Merge Dependabot PR
+      run: gh pr comment --body "@dependabot squash and merge" "$PR_URL"
+      env:
+        PR_URL: ${{ github.event.pull_request.html_url }}
+        GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
```

Comment:
Let's create a new token with permissions required for a pull request check (untrusted context).

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/390/commits/996c5a5399a52c2359e631085a9af9baada0b4de.

Reply:
Merged.